articles/hardware/breakout/digital-inputs.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: breakout_digital-inputs
+title: Breakout Board Digital Inputs
+hardware: true
+device: digital inputs
+videoCaption: This visualizes digital input data using Bonsai visualizers. The "Buttons" window shows which buttons are being pressed. The "HasFlags" window shows whether or not certain button are being pressed (in the case of the breakout board example workflow, "Triangle" or "X"). These visualizers correspond to actual button presses which are demonstrated by the bottom-right breakout board inset.
+---
+
+The following excerpt from the [Breakout Board example workflow](xref:breakout_workflow) demonstrates digital inputs functionality by responding to button presses and saves digital inputs data.
+
+::: workflow
+![/workflows/hardware/breakout/digital-inputs.bonsai workflow](../../../workflows/hardware/breakout/digital-inputs.bonsai)
+:::
+
+The <xref:OpenEphys.Onix1.BreakoutDigitalInput> operator generates a sequence of <xref:OpenEphys.Onix1.BreakoutDigitalInputDataFrame>s. Although the digital inputs are sampled at 4 Mhz, these data frames are only emitted when the port status changes i.e. when a pin, button, or switch is toggled. The digital input ports on the Breakout Board operate at a 3.3V logic levels but are also 5V tolerant. In the Breakout Board example workflow, the `BreakoutDigitalInput`'s `DeviceName` property is set to "BreakoutBoard/BreakoutDigitalInput". This links the `BreakoutDigitalInput` operator to the corresponding configuration operator. 
```

Comment:
`"<xref:OpenEphys.Onix1.BreakoutDigitalInputDataFrame>s."` I know this is technically correct, but it looks weird when rendered on the web page

![image](https://github.com/user-attachments/assets/8241f63e-0d9e-41f4-adea-87b83ef3a151)

Reply:
"status changes i.e. when a pin, button, or switch is toggled" might be a nit-pick, but the 'i.e.,' should be wrapped in parentheses. Also, there should be a comma after 'i.e.,' / 'e.g.,' / etc.